q2_feature_table/_summarize/summarize_assets/app/js/child.js

```diff
@@ -0,0 +1,12 @@
+// ----------------------------------------------------------------------------
+// Copyright (c) 2016-2017, QIIME 2 development team.
+//
+// Distributed under the terms of the Modified BSD License.
+//
+// The full license is in the file LICENSE, distributed with this software.
+// ----------------------------------------------------------------------------
+
+
+document.addEventListener('DOMContentLoaded', function() {
+  parent.postMessage(document.body.scrollHeight, '*');
```

Comment:
This isn't quite working like it should for me, I still end up with the sub-page iframe having a scrollbar (on the bottom on normal view, and on the side when in a mobile-like resolution), instead of the visualization iframe. Might have to do with that nesting?